instrumentation/dropwizard-views-0.7/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/dropwizardviews/DropwizardSingletons.java

```diff
@@ -7,15 +7,21 @@
 
 import io.dropwizard.views.View;
 import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.config.Config;
 import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
 
 public final class DropwizardSingletons {
 
   private static final String INSTRUMENTATION_NAME = "io.opentelemetry.dropwizard-views-0.7";
 
+  private static final boolean SUPPRESS_CONTROLLER_SPANS =
+      Config.get()
+          .getBoolean("otel.instrumentation.common.experimental.suppress-controller-spans", false);
```

Comment:
We should somehow have this in the javaagent-api artifact rather than copying into every module I guess. Either a method on `Config` or something like `JavaAgentConfig` with all of them.